installinator/src/peers.rs

```diff
@@ -371,9 +372,16 @@ impl ArtifactClient {
     }
 
     async fn fetch(&self, artifact_id: ArtifactId, sender: FetchSender) {
+        // TODO: should the installinator try and handle artifact kinds that are
+        // currently unknown to it? (Maybe if a destination is also specified?)
+        // If so, we'll want to change this to a string.
```

Comment:
My gut feeling is "no, and if it needs to we need to update installinator to handle the new kind first". I don't think we need to be too anticipatory here (and hopefully I don't regret this comment in hindsight!).

Reply:
Yeah, I think the answer being "no" here is fine. Importantly, unlike wicket, the installinator is part of the destination version, so there's no version skew issues there.

Reply:
Removed the comment.